controller/src/models/errors.ts

```diff
@@ -5,7 +5,7 @@ export class UndefinedAppError extends Error {}
 export class DuplicatedProfileError extends Error {}
 
 export class AriesAgentError extends FeathersError {
-  constructor(message: string | Error, code: number | undefined, data?: any) {
+  constructor(message: string | Error, code: number | undefined, data?: unknown) {
```

Comment:
Was this throwing new  compilation errors?

Reply:
No the linter was complaining. The superclass uses `unknown` for the type.